src/src/tx_main.cpp

```diff
@@ -356,7 +362,15 @@ void ICACHE_RAM_ATTR HandleFHSS()
   // If the next packet should be on the next FHSS frequency, do the hop
   if (!InBindingMode && modresult == 0)
   {
-    Radio.SetFrequencyReg(FHSSgetNextFreq());
+    if (isDualRadio() && config.GetAntennaMode() == 0) // Gemini mode
+    {
+      Radio.SetFrequencyReg(FHSSgetNextFreq(), SX12XX_Radio_1);
```

Comment:
Ignore this, but minor suggestion if you want:
This could become:
```
{
    Radio.SetFrequencyReg(FHSSgetNextFreq(SX12XX_Radio_1), SX12XX_Radio_1);
    Radio.SetFrequencyReg(FHSSgetNextFreq(SX12XX_Radio_2), SX12XX_Radio_2);
}
```
and then `FHSSgetNextFreq()` has the switching logic in it to return the right freq for the right radio.

Reply:
You and I dont have strong feelings about this, but James likes it this way so it stays... and Im lazy.